tensorflow/python/keras/backend_test.py

```diff
@@ -1880,5 +1880,33 @@ def test_get_session_different_graphs(self):
       self.assertIsNot(session, keras.backend.get_session())
 
 
+class ControlOpsTests(test.TestCase):
+
+  @test_util.run_all_in_graph_and_eager_modes
+  def test_function_switch_basics(self):
+    pred = keras.backend.placeholder([], dtype='bool')
+
+    tensor = keras.backend.switch(pred, lambda: array_ops.constant(True),
+                                  lambda: array_ops.constant(False))
+    value = keras.backend.function(pred, tensor)([True])
+    self.assertEqual(True, value)
+    value = keras.backend.function(pred, tensor)([False])
+    self.assertEqual(False, value)
+
+  @test_util.run_all_in_graph_and_eager_modes
```

Comment:
Please remove this annotation, and move it to the class level.

Reply:
eager mode execution with bool has issues, i guess bool does not seems to be working in eager mode. So i have removed the annotation.

Regards
Amit

Reply:
I think it is important to test both eager and graph mode, and eager mode default on in tf 2.0. Please add the annotation ,and we can check why it is failing.

Reply:
> I think it is important to test both eager and graph mode, and eager mode default on in tf 2.0. Please add the annotation ,and we can check why it is failing.

@qlzh727 , thanks alot, i will try to fix this by tomorrow and if i can not then will seek your help, thanks alot for reaching out like this.

Regards
Amit